### PR TITLE
Mk 854

### PR DESCRIPTION
**Pull request by smirfolio**

### CODE REVIEW

File: dist/ng-obiba-mica.js [hunk 1]
```diff
@@ -3671,6 +3674,38 @@ angular.module('obiba.mica.search')
...
+          if(!$scope.options.SearchHelpText && !$cookies.get('hideSearchHelpText')){
+            $scope.options.SearchHelpText = translation['search.help'];
+          }
+          if(!$scope.options.ClassificationHelpText && !$cookies.get('hideSearchHelpText')){
+            $scope.options.ClassificationHelpText = translation['help.classifications'];
+          }
+        });
+      // Close the Help search box and set the local cookies
+      $scope.closeHelpBox = function () {
+        $cookies.put('hideSearchHelpText', true);
```
Comment (ymarcon) [inline dist/ng-obiba-mica.js]: hideSearchHelpText => micaHideSearchHelpText

File: dist/ng-obiba-mica.js [hunk 2]
```diff
@@ -3671,6 +3674,38 @@ angular.module('obiba.mica.search')
...
+        $scope.options.SearchHelpText = null;
+      };
+
+      // Close the Help classification box and set the local cookies
+      $scope.closeClassificationHelpBox = function () {
+        $cookies.put('hideClassificationHelpBox', true);
```
Comment (ymarcon) [inline dist/ng-obiba-mica.js]: hideClassificationHelpBox => micaHideClassificationHelpBox

File: less/ng-obiba-mica.less [hunk 3]
```diff
@@ -350,3 +350,11 @@
     margin-bottom: 0;
   }
 }
+
+.help-text {
```
Comment (ymarcon) [inline less/ng-obiba-mica.less]: why a specific style?

Comment (smirfolio) [inline less/ng-obiba-mica.less]: Unused style removed 
Done...
Please review

**Pull request merged**